ingest-app/src/cmr/ingest/services/ingest_service/service.clj

```diff
@@ -8,7 +8,8 @@
   "Returns service concept with fields necessary for ingest into metadata db
   under :extra-fields."
   [context concept service]
-  (assoc concept :extra-fields {:service-name (:Name service)}))
+  (assoc concept :extra-fields {:service-name (:Name service) 
+                                :long-name (:LongName service)}))
```

Comment:
I checked the database - there is no LONG_NAME field so this doesn't actually result in anything being saved in Oracle.

Reply:
Thank you so much for catching this. I was always puzzled about what should go into the extra-field...  I did look at variable though, and it contains (assoc concept :extra-fields {:variable-name (:Name variable) :measurement (:LongName variable)})), so I thought variable and service should behave the same way... ?